app/lib/katello/resources/cdn/katello_cdn.rb

```diff
@@ -36,12 +36,16 @@ def valid_path?(_path, _postfix)
           true
         end
 
+        def validate!
+          content_view_id && lifecycle_environment_id && organization
```

Comment:
Check for org first since the other 2 depend on org
```suggestion
          organization && content_view_id && lifecycle_environment_id
```